test/boost/commitlog_test.cc

```diff
@@ -612,7 +612,7 @@ SEASTAR_TEST_CASE(test_commitlog_replay_invalid_key){
         auto& table = db.find_column_family("ks", "t");
         auto& cl = *table.commitlog();
         auto s = table.schema();
-        auto& mt = table.active_memtable();
+        replica::memtable& mt = *table.active_memtables().front();
```

Comment:
This works for a single compaction group, but won't it fail with multiple?

I'd like the test to pass with multiple compaction groups.

Reply:
will try to adjust them for multiple groups, or override like you suggested.